include/meta_json_parser/debug_helpers.cpp

```diff
@@ -192,7 +192,7 @@ void describe_table(cudf::io::table_with_metadata& table_with_metadata, bool dum
 	printf("table (with metadata) has %d columns\n", n_cols);
 	for (int col_idx = 0; col_idx < n_cols; col_idx++) {
 		printf("column(%d) name is \"%s\":\n", col_idx,
-		       table_with_metadata.metadata.column_names[col_idx].c_str());
+		       table_with_metadata.metadata.schema_info[col_idx].name.c_str());
```

Comment:
cuDF in new (stable) RAPIDS released has changed its API and this is the new approach. schema info is a table of structs containing strings instead of table of strings

more info [here](https://docs.rapids.ai/api/libcudf/stable/structcudf_1_1io_1_1table__metadata.html#aca88d41ea923a24e060dbd4165d7bccb)